src/app-layer-pop3.c

```diff
@@ -0,0 +1,55 @@
+/* Copyright (C) 2018 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+/**
+ * \file
+ *
+ * \author Alex Savage <alexander.savage@cyber.gc.ca>
+ *
+ * App-layer parser for POP3 protocol
+ *
+ */
+
+#include "suricata-common.h"
+#include "stream.h"
+#include "conf.h"
+
+#include "util-unittest.h"
+
+#include "app-layer-detect-proto.h"
+#include "app-layer-parser.h"
+
+#include "app-layer-pop3.h"
+#include "rust.h"
+
+void RegisterPOP3Parsers(void)
+{
+    SCLogNotice("Registering Rust pop3 parser.");
+    rs_pop3_register_parser();
```

Comment:
this whole file is no longer needed, you should be able to directly call `rs_pop3_register_parser` from where `RegisterPOP3Parsers` is called

Reply:
This was autogenerated by `script/setup-app-layer.py`
But I will remove this file.

Reply:
ya we need to update that, made a note here https://redmine.openinfosecfoundation.org/issues/4939#note-4

feel free to add other feedback on the script there